eslint/cactbot-locale-order.js

```diff
@@ -37,57 +45,59 @@ const ruleModule = {
       url: 'https://github.com/quisquous/cactbot/blob/main/docs/RaidbossGuide.md#trigger-elements',
     },
     fixable: 'code',
-    schema: [],
+    schema: [
+      {
+        'type': 'array',
+        'items': {
+          'type': 'string',
+        },
+      },
+    ],
     messages: {
-      sortKeys: 'Expected locale object keys to be in an order like [en, de, fr, ja, cn, ko]. \'{{thisName}}\' should be before \'{{prevName}}\'.',
+      sortKeys: 'Expected locale object keys ordered like {{expectedOrder}} (\'{{beforeKey}}\' should before \'{{nextKey}}\')',
```

Comment:
```suggestion
      sortKeys: 'Expected locale object keys ordered like {{expectedOrder}} (\'{{beforeKey}}\' should be before \'{{nextKey}}\')',
```

Tiny grammar suggestion.